_build_npm.ts

```diff
@@ -48,13 +47,16 @@ if (flags.h) {
   printHelpText();
 }
 
+await Deno.remove(`${flags.o}`, { recursive: true }).catch((_) => {});
+
 try {
   await emptyDir(flags.o);
 
   await build({
+    importMap: "./deno.json",
     entryPoints: [flags.e],
     outDir: flags.o,
-    compilerOptions: { target: "ES2021" },
+    compilerOptions: { lib: ["ESNext"], experimentalDecorators: true },
```

Comment:
I always get a little unnerved when the target is unnumbered like "ESNext".

When does ESNext stop 😂?

Reply:
![image](https://github.com/user-attachments/assets/c29178bd-2bd2-4a67-9d77-36b42f588ce8)


yea without it the npm build fail missing types on Sets and ErrorOptions